suites/squid/common/regression/single-site-deploy-and-configure.yaml

```diff
@@ -0,0 +1,145 @@
+tests:
+  - test:
+      name: Setup pre-requisites
+      desc: Setup packages and configuration for cluster deployment
+      module: install_prereq.py
+      abort-on-fail: true
+
+  - test:
+      name: Deploy cluster using cephadm
+      desc: Bootstrap and deploy services
+      polarion-id: CEPH-83573713
+      module: test_cephadm.py
+      config:
+        steps:
+          - config:
+              service: cephadm
+              command: bootstrap
+              args:
+                mon-ip: node1
+          - config:
+              service: host
+              command: add_hosts
+              args:
+                attach_ip_address: true
+                labels: apply-all-labels
+          - config:
+              service: osd
+              command: apply
+              args:
+                all-available-devices: true
+          - config:
```

Comment:
can we remove rgw service deployment from here?
which has been added as part of https://github.com/red-hat-storage/cephci/pull/4440

Reply:
done